### ISSUE TITLE
Math.fround not recognized

### ISSUE THREAD

**Issue by kripken** [google/closure-compiler#3141]
Compiling with advanced opts, `Math.fround` is not recognized and is minified into an invalid `Math.a`. A warning is shown,
```
JSC_INEXISTENT_PROPERTY_WITH_SUGGESTION: Property fround never defined on Math. Did you mean round? at line 3 character 32

```

For an example, see [here](https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%252F%252F%2520ADD%2520YOUR%2520CODE%2520HERE%250Afunction%2520hello(name)%2520%257B%250A%2520%2520alert('Hello%252C%2520'%2520%252B%2520name%2520%252B%2520Math.fround(123.456))%253B%250A%257D%250Ahello('New%2520user')%253B%250A%250A)

**Comment by brad4d** [comment]
Created Google internal bug b/119306748

**Comment by brad4d** [comment]
FYI @shicks is actively working to fix this.

**Comment by brad4d** [comment]
Fixed in 5d04b3ccad67e6d2a1a9edcc8f31a7fa2a1ba996
